contracts/BaseGauge.sol

```diff
@@ -79,6 +79,7 @@ abstract contract BaseGauge is IBaseGauge, Ownable, Initializable {
             uint256 remaining = periodFinish - block.timestamp;
             uint256 leftover = remaining * rewardRate;
             rewardRate = leftover / newDuration;
+            periodFinish = block.timestamp + newDuration;
```

Comment:
I would add `periodFinish` in the event.

Reply:
If there is no rewards live it shouldn't update `periodFinish`